c2corg_ui/static/menu-discourse/menu.html

```diff
@@ -0,0 +1,196 @@
+    </div>
+
+    <ul class="footer-nav">
+      <li>
+        <a href="#" url="articles/106727" class="menu-link">
+          <span class="lang en menu-text">Contact</span>
+          <span class="lang fr menu-text">Contact</span>
+          <span class="lang it menu-text">Contatto</span>
+          <span class="lang de menu-text">Kontakt</span>
+          <span class="lang eu menu-text">Kontaktua</span>
+          <span class="lang ca menu-text">Contacte</span>
+          <span class="lang es menu-text">Contacto</span>
+        </a>
+      </li>
+      <span class="bullet">&bull; </span>
+      <li>
+        <a href="#" url="articles/121965" class="menu-link">
+          <span class="lang en menu-text">Credits</span>
+          <span class="lang fr menu-text">Crédits</span>
+          <span class="lang it menu-text">Credits</span>
+          <span class="lang de menu-text">Kredite</span>
+          <span class="lang eu menu-text">Credits</span>
+          <span class="lang ca menu-text">Crèdits</span>
+          <span class="lang es menu-text">Créditos</span>
+        </a></li>
+      <span class="bullet">&bull; </span>
+      <li><a href="#" url="articles/106731" class="menu-link">
+          <span class="lang en menu-text">Terms of use</span>
+          <span class="lang fr menu-text">Conditions d'utilisation</span>
+          <span class="lang it menu-text">Condizioni d'uso</span>
+          <span class="lang de menu-text">Nutzungsbedingungen</span>
+          <span class="lang eu menu-text">Erabilera-baldintzak</span>
+          <span class="lang ca menu-text">Condicions d'ús</span>
+          <span class="lang es menu-text">Condiciones de uso</span>
+        </a></li>
+      <span class="bullet">&bull; </span>
+      <li>
+        <a href="#" url="articles/106728" class="menu-link">
+          <span class="lang en menu-text">Content license</span>
+          <span class="lang fr menu-text">Licences de contenu</span>
+          <span class="lang it menu-text">Licenze dei contenuti</span>
+          <span class="lang de menu-text">Inhalt Lizenz</span>
+          <span class="lang eu menu-text">Edukiaren lizentziak</span>
+          <span class="lang ca menu-text">Llicències del contingut</span>
+          <span class="lang es menu-text">Licencias de los contenidos</span>
+        </a>
+      </li>
+    </ul>
+
+    <div class="social-networks">
+      <div class="icons">
+        <a href="https://twitter.com/camptocamporg"><img src="#" url="static/img/social/twitter.png" class="src"></a>
+        <a href="https://plus.google.com/104270548458536561874/posts"><img src="#" url="static/img/social/google-plus.png" class="src"></a>
+        <a href="https://www.facebook.com/camptocamp.org/"><img src="#" url="static/img/social/facebook.png" class="src"></a>
+      </div>
+    </div>
+
+  </footer>
+</aside>
+
+<script>
+    var baseUrl = '//www.demov6.camptocamp.org/';
```

Comment:
Actually I meant "at the top of the file" :P 
So that one may immediately see it and fix it. Else it's hard to figure out there's a ``baseUrl`` set somewhere in the file.